integr8ly/manifests
# move to rhsso 1.6.0

# Reason
1.6.1 is much more reliable deployments on OS4

# Verification
- Delete your current operatorsource if you have one
- Create the following operatorsource:
```
apiVersion: operators.coreos.com/v1
kind: OperatorSource
metadata:
  finalizers:
    - finalizer.operatorsources.operators.coreos.com
  name: integreatly-operators
  namespace: openshift-marketplace
spec:
  authorizationToken: {}
  displayName: Integreatly Operators
  endpoint: 'https://quay.io/cnr'
  publisher: Integreatly Publisher
  registryNamespace: philbrookes
  type: appregistry
```
- Subscribe to integreatly-operator in a namespace
- Create an installation CR with `createNamespaces: true`
- When rhsso is provisioned it should be using the v1.6.1 tag of the operator
- RHSSO should deploy succesfully.
